### ISSUE TITLE
Rev Hud can sometimes vanish when you die

### ISSUE THREAD

**Issue by Notamaniac** [yogstation13/yogstation#3106]
Seems to only have happened to me, but if you die, then get cloned your rev hud can vanish, with relogging not helping things.

**Comment by Ratvar** [comment]
Revs have a chance to deconvert when you hit them on the head with brute weapons, you sure you still was rev?

**Comment by Notamaniac** [comment]
Yeah, confirmed it with an admin and my notes still had the rev objective, I later got deconverted with a loyalty implant
